ironpython-git/PKGBUILD

```diff
@@ -1,19 +1,17 @@
 # Maintainer: Michael Schubert <mschu.dev at gmail>
 
 pkgname=ironpython-git
-pkgver=2.7.2.a2.471.g7be8b73
+pkgver=ac72964d7
```

Comment:
`pkgver` is not valid here. It should be `2.7.7.ac72964d7` (details [here](https://wiki.archlinux.org/index.php/VCS_package_guidelines))